38843.9 <c-38#228> Created logger
38843.9 <c-38#228> appl_initialize was called
38843.9 <c-38#228:3> Logger was updated
38843.9 <c-38#228:3> Started intializing cube lib
38843.9 <c-38#228:3> Finished intializing cube lib with worker, starting to work
38843.9 <c-38#228:3> requestCubes => [58] Buffer size: 0
38843.9 <c-38#228:3> Sent requestCubes signal to root
38843.9 <c-38#228:3> Received 1 cubes from root
38843.9 <c-38#228:3> The main loop continues
38843.9 <c-38#228:3> Started solving a cube
38843.9 <c-38#228:3> Cube failed
38843.9 <c-38#228:3> Added failed cube
38843.9 <c-38#228:3> returnFailedCubes => [58] Buffer size: 3
38843.9 <c-38#228:3> Sent 1 failed cubes to root
38843.9 <c-38#228:3> Received receivedFailedCubes signal from root
38843.9 <c-38#228:3> requestCubes => [58] Buffer size: 0
38843.9 <c-38#228:3> Sent requestCubes signal to root
38843.9 <c-38#228:3> Received 1 cubes from root
38843.9 <c-38#228:3> The main loop continues
38843.9 <c-38#228:3> Started solving a cube
39835.6 <c-38#228:3> appl_interrupt was called
39835.6 <c-38#228:3> Interrupting worker
39835.6 <c-38#228:3> appl_interrupt was called
39835.6 <c-38#228:3> appl_withdraw was called
39835.6 <c-38#228:3> Started withdraw thread
39835.6 <c-38#228:3> STOP (0.01s since last cb)
39835.6 <c-38#228:3> Solving interrupted
39835.6 <c-38#228:3> The main loop continues
39835.6 <c-38#228:3> Leaving the main loop due to an interruption
39835.6 <c-38#228:3> Finished withdrawing lib
39835.6 <c-38#228:3> Enter destructor of CubeWorker
39835.6 <c-38#228:3> Time waiting for messages: 0.000
39835.7 <c-38#228:3> Enter destructor of CubeWorkerInterface
39835.7 <c-38#228:3> Enter destructor of CubeRoot
39835.7 <c-38#228:3> Finished withdraw thread
39898.1 <c-38#228:3> Enter destructor
39898.1 <c-38#228:3> Time working: 991.664
39898.1 <c-38#228:3> Time suspended: 0.000
39898.1 <c-38#228:3> Joined cleanup thread
39898.1 <c-38#228:3> Exit destructor
39898.1 <c-38#228:3> Destructing logger
